tasks/register_htpasswd_path_for_app.yml

```diff
@@ -0,0 +1,4 @@
+- name: Set htpasswd target file name for app
```

Comment:
Can you call this task set: `set_htpasswd_path_for_app.yml` instead?

Reply:
Done